### PR TITLE
Add deprecations and removals for Chrome 89

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: site/en/blog/deps-rems-89/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Deprecations and removals in Chrome 89
+description: >
+  A round up of the deprecations and removals in Chrome 89 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-01-28
+authors:
+  - joemedley
+hero: './dep-rem_480.jpg'
```
Comment (robdodson) [inline site/en/blog/deps-rems-89/index.md:9]: Hey @jpmedley for the hero image would you mind uploading it to our image cdn ([guide](https://developer.chrome.com/docs/handbook/how-to/add-media/)) and copy the image into here.

Here's [an example](https://github.com/GoogleChrome/developer.chrome.com/blame/d0275dc7c05ffaad4680a5b52f95928a46b67aa4/site/en/blog/new-in-chrome-54/index.md#L13) of what it should look like from one of Pete's posts.

I have a PR that I'm working on which explains the process better but I don't think it'll be up until tomorrow.

File: site/en/blog/deps-rems-89/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+alt: >
+  Deprecations and removals hero logo
+tags:
+  - deprecations
+  - removals
+  - chrome-89
+---
+
+{% include 'see-all-dep-rem.md' %}
```
Comment (robdodson) [inline site/en/blog/deps-rems-89/index.md:18]:
```suggestion
{% include 'partials/see-all-dep-rem.md' %}
```

File: site/en/blog/deps-rems-89/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Stop cloning sessionStorage for windows opened with noopener
+
+When a window is opened with noopener, Chrome will no longer clone the
+sessionStorage of its opener; it will instead start an empty sessionStorage
+namespace. This brings Chrome in conformance with the HTML specification.
+
+[Chrome Platform Status](https://www.chromestatus.com/feature/5679997870145536) &#124;
+[Chromium Bug](https://crbug.com/771959)
+
+{% include 'deprecations-policy.md' %}
```
Comment (robdodson) [inline site/en/blog/deps-rems-89/index.md:40]:
```suggestion
{% include 'partials/deprecations-policy.md' %}
```

**Review by robdodson — CHANGES_REQUESTED** [review]

**Comment by petele** [comment]
@robdodson @jpmedley - is this good to go?

**Review by robdodson — APPROVED** [review]

**Comment by robdodson** [comment]
It seems fine to me, did y'all decide not to use a hero image?

**Comment by jpmedley** [comment]
No more hero images.

**Pull request merged**